config/afj-prod.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The server already configures it's own config right? This is only needed if you use the afj-rest cli without importing the startServer etc... method yourself.